### PR TITLE
Removed content reuse from 2 files

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**

### CODE REVIEW

**Review by leifmadsen — CHANGES_REQUESTED** [review]
Had a chat with Joanne today, and we think we have another solution to what I was attempting to do. We're not allowed to have procedures that are only partial (breaks the style guide rules), but I think we've come up with a clever solution that works around those rules, while reducing duplication. Stay tuned!

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc [hunk 1]
```diff
@@ -1,11 +1,27 @@
 [id="creating-an-alert-route-in-alertmanager_{context}"]
 = Creating an alert route in Alertmanager
 
-include::../modules/proc_creating-an-alert-route-common-intro.adoc[]
+[role="_abstract"]
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc:4]:
```suggestion
// The introduction to the files proc_creating-an-alert-route-with-templating-in-alertmanager and creating-an-alert-route-in-alertmanager are identical. If you have changes to make, please make the same changes to both introductions.

[role="_abstract"]
```

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-with-templating-in-alertmanager.adoc [hunk 2]
```diff
@@ -2,9 +2,25 @@
 [id="proc_creating-an-alert-route-with-templating-in-alertmanager_{context}"]
 = Creating an alert route with templating in Alertmanager
 
-include::proc_creating-an-alert-route-common-intro.adoc[]
+[role="_abstract"]
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-with-templating-in-alertmanager.adoc:5]:
```suggestion
// The introduction to the files proc_creating-an-alert-route-with-templating-in-alertmanager and creating-an-alert-route-in-alertmanager are identical. If you have changes to make, please make the same changes to both introductions.

[role="_abstract"]
```

**Review by leifmadsen — APPROVED** [review]
Turns out the method I wanted to use won't work, so this is the appropriate approach based on style guide rules.

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc [hunk 3]
```diff
@@ -80,4 +96,40 @@ receivers:
 - name: 'null'
 ----
 
-include::proc_creating-an-alert-route-common-procedure.adoc[]
+. To verify the configuration is loaded into Alertmanager, create a pod with access to `curl`:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc:101]:
```suggestion

// The following steps are duplicated in proc_creating-an-alert-route-with-templating-in-alertmanager. If you have changes to make, please make the same changes to both files.

. To verify the configuration is loaded into Alertmanager, create a pod with access to `curl`:
```

Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc:101]: I will need to make this change in the other branch as I have already merged this down :)

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-with-templating-in-alertmanager.adoc [hunk 4]
```diff
@@ -77,4 +95,41 @@ route:
   receiver: 'slack'
 ----
 
-include::proc_creating-an-alert-route-common-procedure.adoc[]
+. To verify that the configuration loaded into Alertmanager, create a pod with access to the `curl` command:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-with-templating-in-alertmanager.adoc:98]:
```suggestion

// The following steps are duplicated in creating-an-alert-route-in-alertmanager. If you have changes to make, please make the same changes to both files.

. To verify that the configuration loaded into Alertmanager, create a pod with access to the `curl` command:
```

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc (hunk 3 above)
Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc:101]: I can do this tomorrow Leif!

**Pull request merged**